src/Library/public/usbguard/KeyValueParser.cpp

```diff
@@ -35,10 +35,6 @@
 
 namespace usbguard
 {
-  /**
```

Comment:
Why did you remove this note?

Reply:
Because its included in the hpp file documentation and it was only polluting doxygen generated documentation.